ansible/roles/ansible-tvault-contego-extension/tasks/uninstall_rhel.yml

```diff
@@ -7,9 +7,17 @@
   service: name=tvault-contego state=stopped enabled=no
   ignore_errors: yes
 
-- name: uninstall tvault contego extension packages
+- name: uninstall tvault contego extension packages when using python2
   yum:
      name:
         - tvault-contego
         - puppet-triliovault
      state: absent
+  when: PYTHON_VERSION=="python2"
+
+- name: uninstall tvault contego extension packages when using python3
+  yum:
+     name:
+        - tvault-contego
```

Comment:
here names should be python3-tvault-contego

Reply:
requested changes done